src/nni_manager/core/nnimanager.ts

```diff
@@ -474,6 +477,22 @@ class NNIManager implements Manager {
                     }
                 }
                 break;
+            case SEND_TRIAL_JOB_PARAMETER:
+                const tunerCommand: any = JSON.parse(content);
+                assert(tunerCommand.parameter_index >= 0);
+                assert(tunerCommand.trial_job_id !== undefined);
+
+                const trialJobForm: TrialJobApplicationForm = {
+                    jobType: 'TRIAL',
+                    hyperParameters: {
+                        value: content,
+                        index: tunerCommand.parameter_index
+                    }
```

Comment:
Suggest to decouple Command meta data(like trialJobId, parameter_index..etc) from command content, but since it's not a small refactor, you could do it after merging this PR.

Reply:
Thanks, will do it later.